server/services/prismic.js

```diff
@@ -1,8 +1,9 @@
 import Prismic from 'prismic.io';
 
-function removeParagraphs(string) {
-  return string.replace(/<\/?p>/g, '');
-}
+// TODO: DMC commented this because it wasn't being used
+// function removeParagraphs(string) {
+//   return string.replace(/<\/?p>/g, '');
+// }
```

Comment:
If you do use it at some point, perhaps there is a way to do this before you turn the HTML into a string? the string replace can get pretty slow with long texts.